protocol/src/main/resources/protocol.xml

```diff
@@ -59,7 +59,7 @@
 
   <!-- L1 General Messages 0 - 99 -->
 
-  <sbe:message name="ErrorResponse" id="0" description="Standard error response">
+  <sbe:message name="ErrorResponse" id="10" description="Standard error response">
```

Comment:
Ah, so when the buffer was all 0s, SBE assumes it's an ErrorResponse? Oops :sweat_smile:

Reply:
Correct. It was weird. I looked in the wrong direction for some time trying to find the cause of #3152.